src/Components/App/App.js

```diff
@@ -12,21 +15,25 @@ class App extends Component {
     super(props);
   }
 
-  // componentDidMount = async () => {
-  // const bookList = await getBooks()
-  // const allBooks = bookList.results.books
-  // this.props.getInitialBooks(allBooks)
-  // // .catch(error => console.log('error', error))
-  // // refactor to set user message into state depending on where state will be stored
-  //
-  // }
-
   render() {
     return (
       <main>
         <h1 className='title'>BiblioGoal</h1>
-        <Search />
-        <BookContainer />
+      <nav>
+        <Link to='/'>Home</Link>
+        <Link to='/have-read'>Books I Have Read</Link>
+        <Link to='/to-read'>Books To Read</Link>
+      </nav>
+        <Route exact path='/'>
+          <Search />
+          <BookContainer />
+        </Route>
+        <Route exact path='/have-read'>
+          <HaveRead />
+        </ Route>
+        <Route exact path='/to-read'>
+          <ToRead />
+        </ Route>
```

Comment:
I like this method of wrapping the components in the Route. Thanks for sharing this.